.github/workflows/qemu.yaml

```diff
@@ -6,6 +6,8 @@ on:
       - master
       - 'release/**'
   pull_request:
+  schedule:
+    - cron: '0 0 * * *'  
```

Comment:
Any reason why we don't have the comment here? The comment is very helpful to understand this cryptic format. I'm not sure about repeating it in every file, but I think our options are:
- no comment so we don't need to repeat it
- commnet every file